helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -49,6 +49,10 @@ splunkPlatform:
   # Optional. Default value for `sourcetype` field. For container logs, it will
   # be container name.
   sourcetype: ""
```

Comment:
- If the sourcetype is the same across logs, metrics, and traces, we should stick with only using `sourcetype` and avoid introducing `metricsSourceType` and` tracesSourceType`.
- If the sourcetype can be different for each telemetry type (logs, metrics, and traces), we should migrate sourcetype to logsSourcetype. Then we would have `logsSourcetype`, `metricsSourceType` and `tracesSourceType` accordingly.

I'm not certain which use case is more common, so it might be better to keep things simple (using the first option) if we're unsure which one to choose.

Reply:
I think that we should stick to 1) unless there is a good reason to change it

Reply:
thanks @jvoravong. @sbylica-splunk please see the new diff

Reply:
@jvoravong we finished on our end, can you please review?